.circleci/config_template.yml

```diff
@@ -257,6 +257,10 @@ j8_separate_jobs: &j8_separate_jobs
         requires:
           - start_j8_unit_tests
           - j8_build
+    - j8_unit_tests-oa:
```

Comment:
Nit: Job names should use underscores as separators, as in `j8_unit_tests_oa` and `j11_unit_tests_oa`.

Also, for brevity and consistency with other specialized unit tests such as `j8_utests_compression`, `j8_utests_cdc` and `j8_utests_trie`, they should probably be named `j8_utests_oa` and `j11_utests_oa`.